test/spec/PptTestData.scala

```diff
@@ -233,8 +234,48 @@ trait PptTestData extends RegistrationBuilder with MockAuthAction {
                          )
     )
 
-  protected val partnershipDetailsWithScottishPartnership: PartnershipDetails =
+  protected val scottishPartnershipDetails: PartnershipDetails =
     PartnershipDetails(partnershipType = SCOTTISH_PARTNERSHIP,
+                       partnershipName = Some("Scottish Partnership"),
+                       partnershipBusinessDetails =
+                         Some(
+                           PartnershipBusinessDetails(testSatur,
+                                                      testPostcode,
+                                                      None,
+                                                      Some(registrationDetails)
+                           )
+                         )
+    )
+
+  protected val llpPartnershipDetails: PartnershipDetails =
+    PartnershipDetails(partnershipType = LIMITED_LIABILITY_PARTNERSHIP,
+                       partnershipName = Some("LLP Partnership"),
```

Comment:
should there only be names for General & Scottish partnerships?

Reply:
Yes, I think you're right. I hadn't realised that the partnership name was picked up from the underlying `CompanyProfile` for the corp type partnerships. I'll remove from this test data lest it confuses peeps down the line.